.github/workflows/gemini-pr-review.yml

```
@@ -1,456 +1,225 @@
-          # Extract additional instructions from comment
-          ADDITIONAL_INSTRUCTIONS="$(
-            echo "${COMMENT_BODY}" | sed 's/.*@gemini-cli \/review//' | xargs
-          )"
-          echo "additional_instructions=${ADDITIONAL_INSTRUCTIONS}" >> "${GITHUB_OUTPUT}"
+          GEMINI_API_KEY: ${{ secrets.GEMINI_API_KEY }}
+        run: |
+          set -o pipefail
+
+          # Write advanced prompt to a file to avoid YAML quoting issues
+          cat > prompt.txt <<'PROMPT'
+[Role]
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 121: could not find expected ':'

(syntax-check)